c++11/include/opentracing/stringref.h

```diff
@@ -1,6 +1,12 @@
 #ifndef OPENTRACING_STRINGREF_H
```

Comment:
Sure. I just copied it from Bloomberg's PR. Another option might be to use the string_span from the GSL [library](https://github.com/Microsoft/GSL/blob/master/include/gsl/string_span) or a C++11 version of [GSL](https://github.com/martinmoene/gsl-lite).